kong/plugins/aws-lambda/api-gateway/aws/AWSBasicCredentials.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+  Copyright 2016 Adobe Systems Incorporated. All rights reserved.
```

Comment:
The copyright only states the owner of the copyright. The license itself actually tells you what you can do with it. As long as you do not violate the license all is ok. The tricky part is to make sure you don't violate it....